conjure-java-core/src/integrationInput/java/com/palantir/product/EteService.java

```diff
@@ -29,21 +29,20 @@
 public interface EteService {
     /**
      * foo bar baz.
-     *
      * <h2>Very Important Documentation</h2>
-     *
-     * <p>This documentation provides a <em>list</em>:
-     *
+     * <p>This documentation provides a <em>list</em>:</p>
      * <ul>
-     *   <li>Docs rule
-     *   <li>Lists are wonderful
+     * <li>Docs rule</li>
+     * <li>Lists are wonderful</li>
```

Comment:
huh, at one point I think we had an option to eagerly format javadoc. @dansanduleac do we still have a flag for that?